TradeItIosTicketSDK2/TradeItOrdersViewController.swift

```diff
@@ -90,13 +91,10 @@ class TradeItOrdersViewController: TradeItViewController, TradeItOrdersTableDele
                     activityView.hide(animated: true)
                     self.loadOrders()
                 },
-                onSecurityQuestion: { securityQuestion, answerSecurityQuestion, cancelSecurityQuestion in
-                    self.alertManager.promptUserToAnswerSecurityQuestion(
-                        securityQuestion,
-                        onViewController: self,
-                        onAnswerSecurityQuestion: answerSecurityQuestion,
-                        onCancelSecurityQuestion: cancelSecurityQuestion
-                    )
+                onVerifyUrl: { url, complete1FA in
```

Comment:
The `1FA` naming is a little jarring - `challenge` seems a little clearer. This can totally wait for a follow up when we look to merge though.

Reply:
Maybe `completeTradeChallenge` because if you add just `challenge1FA` or `completeChallenge` to the SDK configuration then it's not clear what it's for.